kolibri/utils/cli.py

```diff
@@ -63,6 +71,8 @@
 # https://github.com/PythonCharmers/python-future/issues/22
 click.disable_unicode_literals_warning = True
 
+provisiondevice_file = os.path.join(KOLIBRI_HOME, "provisiondevice.json")
```

Comment:
maybe `automatic_provision_values.json`?

Reply:
Okay, I renamed it to `automatic_provision.json`, which does indeed sound a bit clearer about what it'll do.